src/Utilities/refinePolyhedron.cc

```diff
@@ -108,7 +108,7 @@ GeomPolyhedron refinePolyhedron(const GeomPolyhedron& poly0,
   const unsigned numVertices0 = verts0.size();
   const unsigned numFaces0 = facetVerts0.size();
   // float g_verts[numVertices0][3];
-  int g_vertsperface[numFaces0];
+  vector<int> g_vertsperface(numFaces0);
```

Comment:
It looks like this was done to avoid variable length arrays, but if we would like to keep it on the stack we could. From what I've read VLAs have become standard in C99 and became an option in C++11 so we could use them in our code if we really wanted.

Reply:
This option/support in C++11 might differ between compilers, so it might be better to avoid using VLAs over all.

Reply:
Removed them because g++ 8.* reports them as warnings (and causes the build to fail if you're using warnings as errors).  So what's the standard really say about VLA's?

Reply:
7am brain read `constexpr` above this, my bad the compiler was correct here about VLA warning.